Server/rainbow-server/Sources/Application/Routes/ScoreRoutes.swift

```diff
@@ -16,10 +16,9 @@ func initializeScoreRoutes(app: App) {
     client = app.services.couchDBService
     
     app.router.get("/entries", handler: getAllEntries)
-    app.router.get("/entries/", handler: getOneEntry)
+    app.router.get("/entries", handler: getOneEntry)
```

Comment:
I take it you figured that the parser will check for the presence of the ID then? This makes sense, just wanna be sure that's what you saw.

Reply:
yes,I tested and all the handler does is looks for id in the path. If you look at the handler function it has anonymousId as first parameter. that means its expecting the id part of URL